specification/http/1.0/openapi.json

```diff
@@ -0,0 +1,456 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Open Tool Calling Standard",
+    "version": "1.0.0",
+    "description": "A standard for calling tools (functions or services) in a distributed system.",
+    "contact": {
+      "name": "Arcade.dev",
+      "url": "https://arcade.dev",
+      "email": "dev@arcade.dev"
+    }
+  },
+  "paths": {
+    "/health": {
+      "get": {
+        "summary": "Check health",
+        "description": "Checks the health status of the server.",
+        "responses": {
+          "200": {
+            "description": "Server is healthy."
+          }
+        }
+      }
+    },
+    "/tools": {
+      "get": {
+        "summary": "List tools",
+        "description": "Returns a list of tool definitions.",
+        "security": [
+          {
+            "bearerAuth": []
+          },
+          {}
+        ],
+        "responses": {
+          "200": {
+            "description": "A list of available tools.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "properties": {
+                    "$schema": {
+                      "type": "string",
+                      "format": "uri"
+                    },
+                    "tools": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/components/schemas/ToolDefinition"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/execute": {
```

Comment:
(Not important right now)

Down the road, we will likely want an SSE variant e.g., 

`/tools/stream` to allow tools to report on progress or send notifications